Sources/Charcoal/Filters/Range/RangeFilterView.swift

```diff
@@ -97,6 +97,7 @@ final class RangeFilterView: UIView {
     }
 
     override func layoutSubviews() {
+        sliderInputView.layoutIfNeeded()
```

Comment:
I'm not sure why `super.layoutSubviews()` isn't called from here, but calling that messes up the snapshot test for iPhone 🤷‍♂️

Reply:
Weird 🤔